### PR TITLE
undbx: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Extract e-mail messages from Outlook Express DBX files.

### CODE REVIEW

**Comment by D3vil0p3r** [comment]
Just look that the tags contain the name of the tool.

**Comment by ikstream** [comment]
This tool has just 31 stars and the last commit was four years ago. I am not sure if we should add this at all.

**Comment by D3vil0p3r** [comment]
> This tool has just 31 stars and the last commit was four years ago. I am not sure if we should add this at all.

Hey @ikstream thank you for your feedback. I'm retrieveing these tools I created PRs from the new Kali Purple in order to have purple/blue team tools on BlackArch.

**Comment by ikstream** [comment]
It's a bit suprising to see a tool with just 31 Github Stars to be added to a kali iso.

File: packages/undbx/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+license=('GPL')
+depends=()
+makedepends=('autoconf' 'automake' 'git')
+source=("git+https://github.com/ZungBang/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```
Comment (noraj) [inline packages/undbx/PKGBUILD:20]:
```suggestion
  git describe --long --tags | sed 's/undbx-//' | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```

File: packages/undbx/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=undbx
+pkgver=undbx.0.21.r3.g5e31c75
```
Comment (noraj) [inline packages/undbx/PKGBUILD:5]:
```suggestion
pkgver=0.21.r3.g5e31c75
```

File: packages/undbx/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+pkgrel=1
+pkgdesc='Extract e-mail messages from Outlook Express DBX files.'
+arch=('any')
+groups=('blackarch' 'blackarch-forensic')
+url='https://github.com/ZungBang/undbx'
+license=('GPL')
+depends=()
+makedepends=('autoconf' 'automake' 'git')
```
Comment (noraj) [inline packages/undbx/PKGBUILD:13]:
```suggestion
makedepends=('git')
```

cf. https://github.com/BlackArch/blackarch/pull/2619

**Review by noraj — CHANGES_REQUESTED** [review]

**Review by noraj — APPROVED** [review]
Tested build, install, exec.

**Pull request merged**